src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -471,39 +468,81 @@ def _post_process(
     upload_env_key: str,
     workspace_id: int,
     load_id: str,
-    obj_tuple: _WSObjTuple,
     upload_dir: str,
     output_dir: str,
```

Comment:
Same for output dir, maybe source_dir? Although that's a bit too close to collections_source_dir

Reply:
👍

Reply:
:+1: